src/pages/Signup/SignUp.js

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import "./SignUp.css"
+
+export const SignUp = () => {
+    return (
+        <main className="main-section d-flex">
+            <div className="form-container border-radius-xs padding-sm">
+                <h2 className="head-lg text-center">SignUp</h2>
+                <label id="first_name" className="form-label text-sm">FirstName: </label>
+                <input
+                    htmlFor="first_name"
+                    type="text"
+                    className="form-field border-radius-xs padding-xs"
+                    placeholder="john"
+                    required
+                />
```

Comment:
suggestion: you can put input element inside the label then you wont have to write `htmlFor`

Reply:
I have done this because one of our MAs tell us to do this type binding implicit